src/EditorFeatures/CSharpTest/EditAndContinue/StatementEditingTests.cs

```diff
@@ -10537,5 +10537,67 @@ public void WithExpression_PropertyValueReorder()
         }
 
         #endregion
+
+        #region Top Level Statements
+
+        [Fact]
+        public void TopLevelStatement_CaptureArgs()
+        {
+            var src1 = @"
+using System;
+
+var x = new Func<string>(() => ""Hello"");
+
+Console.WriteLine(x());
+";
+            var src2 = @"
+using System;
+
+var x = new Func<string>(() => ""Hello"" + args[0]);
+
+Console.WriteLine(x());
+";
+            var edits = GetTopEdits(src1, src2);
+
+            // TODO: allow creating a new leaf closure
```

Comment:
>TODO [](http://example.com/codeflow?start=15&length=4)

What is the change suggested in the TODO? Also, consider logging an issue if it's necessary and referencing the issue, or remove the TODO if the suggested change is not necessary.

Reply:
This is a copy and paste from existing lambda capture tests. @tmat do you know if there is an issue for this?

Reply:
Don't have issue.

Reply:
Thanks, I've filed https://github.com/dotnet/roslyn/issues/54672 and updated the comments.